pkg/storage/export/export/export.go

```diff
@@ -767,6 +842,11 @@ func (ctrl *VMExportController) createExporterPodManifest(vmExport *exportv1.Vir
 		Value: currentTime().Add(deadline).Format(time.RFC3339),
 	})
 
+	tokenSecretRef := ""
+	if vmExport.Status != nil && vmExport.Status.TokenSecretRef != nil {
+		tokenSecretRef = *vmExport.Status.TokenSecretRef
+	}
+
```

Comment:
is it possible that `tokenSecretRef` is blank here? Not sure if we want to create the pod with a blank secret name?

Reply:
I would say it's impossible (handleVMExportToken should always populate `vme.Status.TokenSecretRef`), but still wanted to check before dereferencing the pointer.